src/Rule/Boolean.php

```diff
@@ -39,9 +39,6 @@ class Boolean extends Rule
      */
     public function validate($value)
     {
-        if ($value === true || $value === false) {
-            return true;
-        }
-        return $this->error(self::NOT_BOOL);
+        return is_bool($value) ?: $this->error(self::NOT_BOOL);
```

Comment:
I'm in favor of the `is_bool` check, but I do think the readability got worse. What do you think?